Microsoft.Toolkit.Uwp.SampleApp/SamplePages/PeoplePicker/PeoplePickerPage.xaml.cs

```diff
@@ -0,0 +1,32 @@
+﻿// ******************************************************************
```

Comment:
@michael-hawker Updated the placeholder text to avoid confusing.

Our requirements is to call the [People Graph API](https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/api/user_list_people), it's described as "Retrieve a list of person objects ordered by their relevance to the user, which is determined by the user's communication and collaboration patterns, and business relationships.", you could find more details [here](https://developer.microsoft.com/en-us/graph/docs/concepts/people_example), I also updated the control doc to include the link.